apps/web/src/lib/reference/reference-config.ts

```diff
@@ -59,10 +59,4 @@ export const REFERENCE_TYPE_INFO = {
         key: 'webhook_id',
         name: 'Webhook',
     },
-    cost_data_source: {
-        type: 'cost_data_source',
-        key: 'data_source_id',
-        name: 'Data Source',
-        resourceType: 'cost_analysis.DataSource',
-    },
```

Comment:
issue: we decided to use reference config since it is global reference which is stored in reference store..!